client/platform/desktop/backend/serializers/viame.ts

```diff
@@ -269,12 +282,128 @@ async function parseFile(path: string): Promise<TrackData[]> {
+      }
+      resolve();
+    });
+    writeHeader(stringify, meta);
+    Object.values(data).forEach((track) => {
+      const filters = meta.confidenceFilters || {};
+      if (!options.excludeBelowThreshold
+        || Track.trackExceedsThreshold(track.confidencePairs, filters)
+      ) {
+        const sortedPairs = track.confidencePairs.sort((a, b) => a[1] - b[1]);
+        track.features.forEach((keyframeFeature, index) => {
+          const interpolatedFeatures = [keyframeFeature];
+
+          /* If this is not the final keyframe and interpolation is enabled */
+          if (keyframeFeature.interpolate && keyframeFeature.frame < track.end) {
+            const nextFeature = track.features[index + 1];
+            /* Interpolate all features (a, b) NOT INCLUDING a or b */
+            for (let f = keyframeFeature.frame + 1; f < nextFeature.frame; f += 1) {
+              const interpolated = Track.interpolate(f, keyframeFeature, nextFeature);
+              if (interpolated === null) {
+                throw new Error('null interpolated track should be impossible here');
+              }
+              interpolatedFeatures.push(interpolated);
+            }
+          }
+
+          /* Iterate all features (real and interpolated) */
+          interpolatedFeatures.forEach((feature) => {
+            /* Column 2 is timestamp in video, image name in image sequence */
+            let column2 = '';
+            if (meta.type === 'image-sequence') {
+              column2 = meta.originalImageFiles[feature.frame];
+            } else if (meta.type === 'video') {
+              column2 = moment.utc(feature.frame / meta.fps).format('H:M:S.SSS');
+            }
+
+            const row = [
+              track.trackId,
+              column2,
+              feature.frame,
+              ...(feature.bounds as number[]),
+              sortedPairs[0][1],
+              feature.fishLength || -1,
+              ...flattenDeep(sortedPairs),
+            ];
+
+            /* Feature Attributes */
+            Object.entries(feature.attributes || {}).forEach(([key, val]) => {
+              row.push(`${AtrToken} ${key} ${val}`);
+            });
+
+            /* Track Attributes */
+            Object.entries(track.attributes).forEach(([key, val]) => {
+              row.push(`${TrackAtrToken} ${key} ${val}`);
+            });
+
+            /* Geometry */
+            if (feature.geometry && feature.geometry.type === 'FeatureCollection') {
+              feature.geometry.features.forEach((geoJSONFeature) => {
+                if (geoJSONFeature.geometry.type === 'Polygon') {
+                  const coordinates = flattenDeep(geoJSONFeature.geometry.coordinates[0]);
+                  row.push(`${PolyToken} ${coordinates.map(Math.round).join(' ')}`);
+                } else if (geoJSONFeature.geometry.type === 'Point') {
+                  if (geoJSONFeature.properties) {
+                    const kpname = geoJSONFeature.properties.name;
```

Comment:
```suggestion
                    const kpname = geoJSONFeature.properties.key;
```
Fairly certain that should be `.key` instead of `.name`
https://github.com/VIAME/VIAME-Web/blob/master/server/viame_server/serializers/viame.py#L290

Reply:
Well spotted.